src/main/kotlin/no/nav/pdfgen/Utils.kt

```diff
@@ -58,7 +58,9 @@ fun createPDFA(imageStream: InputStream, outputStream: OutputStream) {
         document.addPage(page)
         val image = toPortait(ImageIO.read(imageStream))
 
-        val pdImage = LosslessFactory.createFromImage(document, image)
+        val quality = 1.0f
```

Comment:
Kan justere denne ned til standardinnstillingen på 0.75f for å redusere filstørrelsen ytterligere.